python/instrumentation/openinference-instrumentation-smolagents/README.md

```diff
@@ -14,42 +14,57 @@ pip install openinference-instrumentation-smolagents
 
 ## Quickstart
 
-This quickstart shows you how to instrument your guardrailed LLM application 
+This quickstart shows you how to instrument your LLM agent application.
```

Comment:
I noticed this copy/paste elsewhere, so will try to scrub for that if I end up nudging another package.